src/lang/std/artifactGraph.ts

```diff
@@ -722,10 +729,12 @@ export function expandSegment(
     { key: segment.pathId, types: ['path'] },
     artifactGraph
   )
-  const surf = getArtifactOfTypes(
-    { key: segment.surfaceId, types: ['wall'] },
-    artifactGraph
-  )
+  const surf = segment.surfaceId
+    ? getArtifactOfTypes(
+        { key: segment.surfaceId, types: ['wall'] },
+        artifactGraph
+      )
+    : undefined
```

Comment:
In the event of no surface _should_ it return undefined?

Reply:
Yes. It means that a segment hasn't been extruded.